docs/project/library-servicing.md

```diff
@@ -29,9 +29,22 @@ If this library ships both inbox on a platform and in its own library package th
 ```
 Where the `AssemblyVersion` is set to the old version before updating. To determine if the library ships inbox you can look at for `InboxOnTargetFramework` item groups or `TreatAsOutOfBox` suppressions in the pkgproj for the library.
 
+If the library is part of a Aspnetcore or .NET targeting pack then we cannot increment the assembly version. For Aspnetcore and WindowsDesktop, You can examine the [refpack-assemblies.md](refpack-assemblies.md) 
```

Comment:
Is that a universal statement for 2.x, 3.x and 5.0? Is the list of targeting pack assemblies the same for all these releases?

Reply:
This is true for 5.0. I dont think we need this for 2.1 and 3.1 cc @ericstj @dougbu

Reply:
We have fewer guards in place in 2.1 and 3.1. Our pinning of the relevant packages in 3.1 (see https://github.com/dotnet/aspnetcore/blob/release/3.1/eng/SharedFramework.External.props#L9-L19) is enough because (fortunately or unfortunately) because https://github.com/dotnet/aspnetcore/blob/release/3.1/src/Framework/test/TestData.cs#L152-L285 is only used to check our targeting pack content.

Bottom line, assembly versions in 2.1 and 3.1 can probably change just fine; doesn't hurt for them to be consistent. We would however need to react if a ref/ assembly that goes into our targeting packs needed servicing and its assembly version changed.